include/mbedtls/cipher.h

```diff
@@ -69,102 +69,102 @@ extern "C" {
 #endif
 
 /**
- * \brief     An enumeration of supported ciphers.
+ * \brief     Supported cipher types.
  *
  * \warning   ARC4 and DES are considered weak ciphers and their use
- *            constitutes a security risk. We recommend considering stronger
- *            ciphers instead.
+ *            constitutes a security risk. We recommend you consider using
```

Comment:
I'd rather use an impersonal form here. Suggestion:
Considering stronger ciphers instead is recommended.

Reply:
"Arm recommends" is as impersonal as documentation gets. Changed accordingly.

Reply:
The text is fine by me.